bigtop-deploy/puppet/manifests/python.pp

```diff
@@ -20,4 +20,12 @@
           target => '/usr/bin/python2',
         }
     }
+    
+    # BIGTOP-4069 create the soft link of python for openEuler smoke-tests
```

Comment:
We only need comment like this if the code is complex or counter intuitive. Please remove it in this case.